src/schema/schema.js

```diff
@@ -212,95 +145,53 @@ const typeDefs = gql`
     postedBy: ID!
   }
 
-  type CreateJobPostingResponse {
-    jobPosting: JobPosting!
-  }
-
-  input AddJobApplicationToJobPostingInput {
-    jobApplicationId: ID!
-    jobPostingId: ID!
-  }
-
-  type AddJobApplicationToJobPostingResponse {
-    jobPosting: JobPosting!
-  }
-
-  type DeleteJobPostingResponse {
-    deletedJobPostingId: ID!
-  }
-
-  input CreateSkillInput {
-    name: String!
-  }
-
-  type CreateSkillResponse {
-    skill: Skill!
-  }
-
-  type DeleteSchoolResponse {
-    deletedSchoolId: ID!
-  }
-
   type LoginUserResponse {
     token: String
     user: User
   }
 
   type Query {
-    getUserById(id: ID!): User
+    getUserById(userId: ID!): User
```

Comment:
I don't really like that we changed `id` to `userId`, `jobAppId`, etc. Was there a reason behind this?

In an input type where there are multiple ids, this naming makes sense because we have to differentiate between them. In an input where there's just a single id, I would prefer to leave it as id

Reply:
I'm not sure if we can do it like this because it was giving me errors when I tried to change the name from userId to id. It may have something to do with Prisma.